assets/railcraft/lang/en_US.lang

```diff
@@ -1232,6 +1232,8 @@ track_kit.railcraft.railcraft_gated.tips=Track with built-in Gate\n§o§c- Apply
 track_kit.railcraft.railcraft_gated_one_way.name=Gated One-Way
 track_kit.railcraft.railcraft_gated_one_way.tips=Track with built-in One-Way Gate\n§o§c- Apply Redstone to open -
 
+track_kit.railcraft.railcraft_junction.name=Junction
+
```

Comment:
See my first commit message. In 1.7, it didn't have tips for the kit but only for the track type, so I opened this pull request for feedbacks.